tests/Main/ReflectionTests.fs

```diff
@@ -172,8 +172,8 @@ type TestRecord = {
 }
 
 type TestUnion =
-    | StringCase of String: string * string
-    | IntCase of Int: int
+    | StringCase of SomeString: string * string
```

Comment:
I didn't want to risk some coincidental equality of the field name and the (pascal cased version of?) the type, so made the field names clearly distinct.

Reply:
LGTM. This code is just for testing purposes.